pkgs/development/python-modules/python-bsblan/default.nix

```
@@ -17,33 +18,35 @@
 
 buildPythonPackage rec {
   pname = "python-bsblan";
-  version = "0.5.18";
-  format = "pyproject";
+  version = "0.6.00";
+  pyproject = true;
 
   disabled = pythonOlder "3.9";
```

```suggestion
  disabled = pythonOlder "3.12";
```
https://github.com/liudger/python-bsblan/blob/v0.6.2/pyproject.toml#L28